db/db_mysql/migrations/20191022184235_0.9.0_imap.sql

```diff
@@ -0,0 +1,8 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE IF NOT EXISTS `imap` (user_id bigint,host varchar(255),port int,username varchar(255),password varchar(255),modified_date datetime,tls boolean,enabled boolean,folder varchar(255,restrict_domain varchar(255),delete_reported_campaign_email boolean,last_login datetime,imap_freq int);
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS `imap` (user_id bigint,host varchar(255),port int,username varchar(255),password varchar(255),modified_date datetime,tls boolean,enabled boolean,folder varchar(255),restrict_domain varchar(255),delete_reported_campaign_email boolean,last_login datetime,imap_freq int);
```

It looks like you're missing a parenthesis here causing the SQL to be invalid.